app/controllers/admin_shelters_controller.rb

```diff
@@ -0,0 +1,8 @@
+class AdminSheltersController < ApplicationController
```

Comment:
I still think this is supposed to live within a "admin" subdirectory of controllers and just be called SheltersController.  We can leave it as is for now but will want to refactor later I think.